docs/extensions/installing-extensions.md

```diff
@@ -0,0 +1,161 @@
+---
+contributors: Esther Agbaje
+description: Discover how to install extensions into your Directus project.
+---
+
+# Installing Extensions
+
+There are three possible ways to install extensions to your Directus instance:
+
+1. Installing through NPM
+2. Installing through the Extensions Folder
+3. Installing through the Marketplace
+
+## Installing through NPM
+
+Package extensions are essentially npm packages that can be installed from the npm registry. Before you begin, ensure
+you have a [self-hosted instance of Directus](/self-hosted/quickstart) via Docker installed on your system.
+
+**1. Modify docker-compose.yml**
```

Comment:
This needs a bit more context with perhaps some more steps to make work.

Reply:
I don't get what you mean here

Reply:
Where does this modification go?

Reply:
This modification happens in the `docker-compose.yml` file under the build command. Is that what you mean?